_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,197 @@
+      own funds, but it'd be better to avoid leaking evidence in the
+      first place.  Fournier [proposed][oblivious state acquisition]
+      using a cryptographic primitive called [oblivious transfer][] to
+      allow you to get the latest state from your counterparty without
+      revealing whether or not you plan to use it.  FIXME:harding maybe
+      write more or delete item, see what Fournier replies to my post on list
+
+- **New PSBT version proposed:** Andrew Chow, author of the [BIP174][]
+  specification of [Partially Signed Bitcoin Transactions][topic psbt]
+  (PSBTs), [proposed][psbt2] a new version of PSBTs that will contain
+  several backwards incompatible features, although it'll be largely the
+  same as the current version 0 PSBT.
+
+FIXME:harding maybe write something about BIP155; see what jnewbery
+writes in the notable code changes section first
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update
+
+- [Bitcoin Core 0.21.0rc3][Bitcoin Core 0.21.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20564][] and [#20612][Bitcoin Core #20612] Don't send
+  'sendaddrv2' to pre-70016 software, and send before 'verack'
+  FIXME:jnewbery, maybe also mention mailing list post and #20599
+
+- [Bitcoin Core #19776][] net, rpc: expose high bandwidth mode state via getpeerinfo FIXME:moneyball
+
+- [Bitcoin Core #19858][] adds a new method for finding high-quality peers with
+  the goal of making [eclipse attacks][topic eclipse attacks] more difficult.
+  It opens an outbound connection every five minutes on average to a new peer
+  and syncs headers with it. If the peer tells the node about new blocks, the
+  node disconnects an existing block-relay-only peer and gives that connection
+  slot to the new peer; otherwise, the new peer is dropped.  Provided the node
+  knows at least one honest peer, this peer rotation should raise the cost of
+  sustaining a partitioning attack against it, as the node should always
+  eventually find the valid chain with the most work. The increased rotation and
+  security isn't free---the change implies a reduction in the number of open
```

Comment:
use `aren't` instead of `isn't` because there is a plural compound subject in (rotation, security)
```suggestion
  security aren't free---the change implies a reduction in the number of open
```